### PR TITLE
DAG-2015 Add aborted field to Version object in evergreen.py

### PR DESCRIPTION
**Pull request by zituo-jin**

### CODE REVIEW

File: .evergreen.yml [hunk 1]
```diff
@@ -26,7 +26,13 @@ functions:
...
         pip install --upgrade pip
 
-        pip install poetry
+        pip install pipx
+
+        export PIPX_HOME="$HOME/pipx"
+        export PIPX_BIN_DIR="$HOME/pipx/bin"
+        export PATH="$PATH:$PIPX_BIN_DIR"
+
+        pipx install poetry
```
Comment (zamj) [inline .evergreen.yml:35]: Can you pin pipx and poetry like we do [here](https://github.com/10gen/build-baron-tools/blob/master/.evergreen.yml#L85)? That way we can better insure against breakages from new versions of them.

**Review by zamj — APPROVED** [review]
LGTM mod my comment

File: .evergreen.yml (hunk 1 above)
Comment (zituo-jin) [reply .evergreen.yml]: done

**Comment by zituo-jin** [comment]
evergreen merge

**Comment by zituo-jin** [comment]
evergreen merge

**Pull request merged**